classes/form/smartmenu_item_edit_form.php

```diff
@@ -704,6 +710,13 @@ public function validation($data, $files) {
             }
         }
 
+        if ($data['type'] == smartmenu_item::TYPEMAILTO) {
+            // Verify that the email field is not empty.
+            if (empty($data['email'])) {
+                $errors['email'] = get_string('required');
```

Comment:
Given the fact that the user might not understand that he should just add the email adress without a "mailto:" prefix and that the form will clean all inputs which are not valid email addresses, I would propose to improve the error message in this case to `Required – Please enter a valid email address without the \'mailto:\' prefix.`

I will change that in a review commit